loader/CMakeLists.txt

```diff
@@ -125,6 +125,61 @@ if (WIN32)
 	set(SOURCES ${SOURCES} ${CMAKE_CURRENT_BINARY_DIR}/info.rc)
 endif()
 
+
+if (ANDROID AND GEODE_ANDROID_USE_BREAKPAD)
+	CPMAddPackage(
+		NAME "Breakpad"
+		VERSION "2023.06.01"
+		GIT_REPOSITORY "https://chromium.googlesource.com/breakpad/breakpad.git"
+	)
+
+	if (Breakpad_ADDED)
+		include(FetchContent)
+
+		FetchContent_Declare(lss
+				GIT_REPOSITORY https://chromium.googlesource.com/linux-syscall-support.git
+				GIT_TAG v2022.10.12
+				SOURCE_DIR ${Breakpad_BINARY_DIR}/src/third_party/lss
+		)
+
+		FetchContent_MakeAvailable(lss)
+
+			add_library(Breakpad STATIC
```

Comment:
y the random indentation increase

Reply:
lol oops 😭 must've been a bad copy paste